.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
```

Comment:
- It's important to ensure that the code follows RESTful architecture. Consider organizing the code into separate endpoints for better structure and readability.